### ISSUE TITLE
News18 Kannada News channel not streaming

### ISSUE THREAD

**Issue by giridathac** [krreet/plugin.video.jiotv#86]
For sometime(probably few months) now News18 Kannada News channel hasn't been streaming. Can somebody look??

**Comment by giridathac** [comment]
Agree and thank you for the response. So, it will not be available on JioTV??

**Comment by technofreak24** [comment]
> Agree and thank you for the response. So, it will not be available on JioTV??

Yes it will not be available till the fix is done. The problem is in KODI Addon and not the source. The addon developer has to fix the code.
In the official Reliance JioTV App, News 18 Kannada is working perfectly okay now.

**Comment by technofreak24** [comment]
**News 18 Kannada** works fine in JioTV KODI Addon only on Channel URL API v2.1 in Addon constants.py
It will not work on API v2.0
![IMG_20230918_N18K2_055722](https://github.com/krreet/plugin.video.jiotv/assets/121749085/7adfe6a3-d225-48e1-9f84-414c7f089dcc)
![IMG_20230918_N18K1_055733](https://github.com/krreet/plugin.video.jiotv/assets/121749085/adf0095b-5e7b-4dd5-b465-38ef32f846d7)

**Comment by technofreak24** [comment]
Attaching updated zip addon to this comment 
[plugin.video.jiotv-2.3.9~SonyCNHDTamilFix.zip](https://github.com/krreet/plugin.video.jiotv/files/12655896/plugin.video.jiotv-2.3.9.SonyCNHDTamilFix.zip)
Or from Discord server on https://discord.com/channels/1069583196326469634/1069583197186293774/1153546088243007578

**Comment by giridathac** [comment]
Thank you.

**Comment by giridathac** [comment]
Yes, saw that. Thank you for the update.

I still see a small problem in News18 Kannada channel not playing from IPTV playlist (TV tab instead of Addon Tab in Kodi). Rest channels play wonderfully well from TV tab. One thing I noticed is you can play programs telecasted earlier but not the current program through s
From addon tab I can see News18 works well. Hope you can correct this also.

**Comment by technofreak24** [comment]
> @giridathac News18 Kannada channel not playing from IPTV playlist (TV tab instead of Addon Tab in Kodi)

The IPTV Playlist has to be updated from JioTV KODI Addon Settings. It reads "Generate Playlist on Startup"...Once updated by setting ON, it can be turned OFF to avoid duplicate channels issue.

**Comment by giridathac** [comment]
Let me try and get back. Thank you..

**Comment by giridathac** [comment]
Hi,

I did try to generate playlist at start with the new zip file.

I can't do a switch live to News18 Kannada channel, but can play programme
telecasted at any particular time earlier. I have this problem with News18
Kannada channel among Kannada channels.

Regards,
Giri
